spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
 
-describe LogStash::Inputs::EventLog, :windows => true do
+describe "LogStash::Inputs::EventLog", :windows => true do
+
+  before(:all) do
+    require 'logstash/inputs/eventlog'
```

Comment:
as I said, putting back the require call in the register method would fix this. we can open a separate issue for this if we want to explore that again.